dashboard/deploy_dashboard_gcloud

```diff
@@ -26,8 +50,16 @@ EOF
 
 PUBSUB_FUNCTIONS="udmi_target udmi_state udmi_config udmi_reflect"
 for func in $PUBSUB_FUNCTIONS; do
-    echo Deploying pubsub-trigger function $func...
-    gcloud functions deploy $func --set-env-vars GCP_PROJECT=$PROJECT --trigger-topic=$func --runtime=$RUNTIME --project=$PROJECT --source=$SOURCE "$@" &
+    echo Deploying pubsub-trigger function $func as ${!func}...
```

Comment:
whyfor the ${!}?

Reply:
The functions (entry points) in `index.js` dont change name, but the triggers and name of the instance of deployed cloud function do. So, for udmi_target, we need the function name `$func` and the trigger/instance name `$udmi_target`, so using `${!func}` to access `$udmi_target`, `$udmi_state` etc programmatically within the loop